content/documentation/overview/index.adoc

```diff
@@ -104,7 +104,11 @@ The Istio Config menu entry displays a list of all of the available Istio config
 .List of Istio Configs
 image::istio-list.png[Demo Website]
 {nbsp} +
-You can inspect the configuration yaml around a specific Istio object.
+You can view and edit and delete the configuration yaml around a specific Istio object.
+{nbsp} +
```

Comment:
Remove and by ,
`You can view and edit and delete the configuration yaml around a specific Istio object.`
by
`You can view , edit and delete the configuration yaml around a specific Istio object.`